### ISSUE TITLE
Issue a warning about ill-formed "--trace xxxx" options

### ISSUE THREAD

**Issue by SegundoBob** [leo-editor/leo-editor#3351]
### Severity
This is a low severity bug.  It is a minor inconvenience for me.

### Bug Desciption
"--trace xxxx" is a Leo-Editor command-line option.

If you try to open one Leo-Editor file using a command line containing the "--trace xxxx" command-line option, Leo-Editor opens xxxx.leo and then the Leo-Editor file you intended it to open.   The focus is left on the xxxx.leo tab.

**Comment by SegundoBob** [comment]
You should also correct the command line help which shows a space instead of "=":

```
--trace TRACE-KEY     add one or more strings to g.app.debug. One or more of...
                        abbrev, beauty, cache, coloring, drawing, events, focus, git, gnx
                        importers, ipython, keys, layouts, plugins, save, select, sections,
                        shutdown, size, speed, startup, themes, undo, verbose, zoom

```

Regards,
SegundoBob

**Comment by edreamleo** [comment]
@SegundoBob Good catch.

**Comment by edreamleo** [comment]
@SegundoBob I am going to close this issue and mark it `Can'tFix` (with any reasonable amount of work).

I'll consider a PR that fixes this issue cleanly (without an heroic amount of work).

**Comment by edreamleo** [comment]
Closed via #3381.
